data-pipeline-flink/dp-core/src/main/scala/org/sunbird/dp/core/cache/DataCache.scala

```diff
@@ -131,7 +131,7 @@ class DataCache(val config: BaseJobConfig, val redisConnect: RedisConnect, val d
     try {
       set(key, value);
     } catch {
-      case ex@(_: JedisConnectionException | _: JedisException) =>
+      case ex: JedisException =>
```

Comment:
`JedisConnectionException` class extends from JedisConnection. We might have added it to specifically capture a JedisConnectionException. Is this change necessary?

Reply:
`ex@(_: JedisConnectionException | _: JedisException)` in this case was present in the original commit, I had removed it to make it consistent, but I see that this change is really not necessary. Let me know, can leave this included in this PR or can leave it out.

Reply:
modification removed from the PR